pkg/windows/windows.go

```diff
@@ -498,14 +483,22 @@ func (vm *windows) ConfigureWICD(apiServerURL string, credentials *Authenticatio
 	}
 	wicdServiceArgs := fmt.Sprintf("controller --windows-service --log-dir %s --api-server %s --sa-ca %s --sa-token %s",
 		wicdLogDir, apiServerURL, wicdCAFile, wicdTokenFile)
-	wicdService, err := newService(wicdPath, wicdServiceName, wicdServiceArgs, nil)
+	wicdService, err := newService(wicdPath, WicdServiceName, wicdServiceArgs, nil)
 	if err != nil {
-		return errors.Wrapf(err, "error creating %s service object", wicdServiceName)
+		return errors.Wrapf(err, "error creating %s service object", WicdServiceName)
 	}
 	if err := vm.ensureServiceIsRunning(wicdService); err != nil {
-		return errors.Wrapf(err, "error ensuring %s Windows service has started running", wicdServiceName)
+		return errors.Wrapf(err, "error ensuring %s Windows service has started running", WicdServiceName)
+	}
+	vm.log.Info("configured", "service", WicdServiceName, "args", wicdServiceArgs)
+	return nil
+}
+
+func (vm *windows) DeconfigureWICD() error {
+	wicdSvc := &service{name: WicdServiceName}
+	if err := vm.ensureServiceIsRemoved(wicdSvc); err != nil {
+		return errors.Wrapf(err, "could not remove service %s", WicdServiceName)
```

Comment:
We aren't always removing the WICD service (if it doesn't exist already for example) so the error msg should reflect that
e.g. `unable to ensure WICD service is removed`

Reply:
This error is stuttering, if you look at the errors returned by ensureServiceIsRemoved